lib/seadsa/RemovePtrToInt.cc

```diff
@@ -232,8 +232,12 @@ class PointerPromoter : public InstVisitor<PointerPromoter, Value *> {
 
   Value *visitSub(BinaryOperator &I) {
     // errs() << "visitSub: " << I << "\n";
-    // XXX TODO
-    return nullptr;
+    if (!isa<ConstantInt>(I.getOperand(1))) {
+      // TODO: a non-constant operand
+      return nullptr;
+    }
+
+    return visitAdd(I);
```

Comment:
this is only correct if the value of `I` is 0. Unless you change `visitAdd` to expect a Sub and multiply the value by -1

Reply:
Yeah I was going to close the PR